ee/clickhouse/models/event.py

```diff
@@ -39,19 +41,19 @@ def create_event(
     if elements and len(elements) > 0:
         elements_chain = elements_to_string(elements=elements)
 
-    sync_execute(
-        INSERT_EVENT_SQL(),
-        {
-            "uuid": str(event_uuid),
-            "event": event,
-            "properties": json.dumps(properties),
-            "timestamp": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
-            "team_id": team.pk,
-            "distinct_id": str(distinct_id),
-            "elements_chain": elements_chain,
-            "created_at": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
-        },
-    )
+    data = {
+        "uuid": str(event_uuid),
+        "event": event,
+        "properties": json.dumps(properties),
+        "timestamp": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
+        "team_id": team if isinstance(team, int) else team.pk,
+        "distinct_id": str(distinct_id),
+        "elements_chain": elements_chain,
+        "created_at": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
+        # TODO: Support persons on events
+    }
+    p = ClickhouseProducer()
+    p.produce(topic=KAFKA_EVENTS_JSON, sql=INSERT_EVENT_SQL(), data=data)
```

Comment:
👍  
The only concern I'd have with this is that it could be slightly more annoying to debug, e.g. wrote a bad event, but I'll take the speed improvements + looks like we already use it for persons and other stuff too.
Noticed that in the `bulk_create_events` we still have `sync_execute` below that's probably fine and potentially faster that way.